easybuild/easyconfigs/h/hyperspy/hyperspy-1.1.1-intel-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,72 @@
+easyblock = 'Bundle'
+
+name = 'hyperspy'
+version = '1.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://scikit-learn.org/stable/index.html'
+description = """Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
+building upon numpy, scipy, and matplotlib. As a machine-learning module,
+it provides versatile tools for data mining and analysis in any field of science and engineering.
+It strives to be simple and efficient, accessible to everybody, and reusable in various contexts."""
+
```

Comment:
Same here - shouldn't this be the HyperSpy description, rather than the scikit-learn one?